sui_programmability/examples/math/sources/ecdsa.move

```diff
@@ -26,7 +26,7 @@ module math::ecdsa {
     public entry fun keccak256(data: vector<u8>, recipient: address, ctx: &mut TxContext) {
```

Comment:
Irrelevant to this PR, but why this keccak function lives here under ECDSA?

Reply:
yeah we want to put it in something like hash.move

Reply:
https://github.com/MystenLabs/sui/pull/5149